help/restrict-moving-messages.md

```diff
@@ -61,6 +84,26 @@ Permissions for moving messages between channels can be configured separately.
 
 {end_tabs}
 
+## Configure who can move messages to another channel from a specific channel
```

Comment:
This seems OK, but I do wonder if we can simplify to just "Configure who can move messages currently present in a specific channel".